src/mbgl/gl/attribute.hpp

```diff
@@ -255,8 +266,9 @@ class Attributes {
     }
 
     template <class DrawMode>
-    static Bindings bindings(const VertexBuffer<Vertex, DrawMode>& buffer) {
```

Comment:
Does anything actually fail if you don't make this change? In theory, this function is called only for active attributes. If it's called for an attribute that isn't actually present, then there's a bug somewhere I think.